File: tuan2/PhanB/bai43.py
import sys

def fib_lap(n):
    if n <= 2:
        return 1
    a, b = 1, 1
    for _ in range(3, n + 1):
        a, b = b, a + b
    return b

def main():
    line = sys.stdin.readline().strip()
    if not line:
        return
    n = int(line)
    if n > 92:
        print("tran long long")
    else:
        print(f"F = {fib_lap(n)}")

if __name__ == '__main__':
    main()

File: tuan2/PhanB/bai43.jl
function fib_lap(n::Int)
    if n <= 2
        return Int64(1)
    end
    a, b = Int64(1), Int64(1)
    for _ in 3:n
        a, b = b, a + b
    end
    return b
end

function main()
    line = readline()
    if isempty(strip(line))
        return
    end
    n = parse(Int, strip(line))
    if n > 92
        println("tran long long")
    else
        println("F = $(fib_lap(n))")
    end
end

main()